docs/reference/api/storage/MBRBlockDevice.md

```diff
@@ -1,2 +1,17 @@
 ## MBRBlockDevice
 
+The MBRBlockDevice class provides a way to manage a Master Boot Record (MBR) on a storage device allowing for partitioning and formatting. The operating systems use the MBR to load the storage device properly into memory.
```

Comment:
Query: To make sure I'm reading this correctly, is the first sentence saying that without the ability to manage an MBR, you can't partition and format? Also, what can't you partition and format?